velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -25,6 +25,23 @@ namespace facebook::velox::substrait {
 /// This class is used to convert the Substrait plan into Velox plan.
 class SubstraitVeloxPlanConverter {
  public:
+  struct SplitInfo {
+    /// The Partition index.
+    u_int32_t partitionIndex;
+
+    /// The file paths to be scanned.
+    std::vector<std::string> paths;
+
+    /// The file starts in the scan.
+    std::vector<u_int64_t> starts;
+
+    /// The lengths to be scanned.
+    std::vector<u_int64_t> lengths;
+
+    /// The file format of the files to be scanned.
+    int32_t fileFormat;
```

Comment:
What is the meaning of the integer here? Should we use dwio::common::FileFormat enum instead?

In the Substrait proto, I'm seeing file format defined as a struct:

https://github.com/substrait-io/substrait/blob/main/proto/substrait/algebra.proto#L118

```
      oneof file_format {
        ParquetReadOptions parquet = 9;
        ArrowReadOptions arrow = 10;
        OrcReadOptions orc = 11;
        google.protobuf.Any extension = 12;
      }
```

Reply:
oneof file_format {
        ParquetReadOptions parquet = 9;
        ArrowReadOptions arrow = 10;
        OrcReadOptions orc = 11;
        google.protobuf.Any extension = 12;
      }
The above code is newly committed in [PR#169](https://github.com/substrait-io/substrait/pull/169). And the FileFormat is described as follow in velox.

      enum FileFormat{
        FILE_FORMAT_UNSPECIFIED = 0;
        FILE_FORMAT_PARQUET  = 1;
      }

I think we can change this after applying [PR#169](https://github.com/substrait-io/substrait/pull/169).

Reply:
Got it. Let's change int32_t to dwio::common::FileFormat in this PR and update Substrait proto in a follow-up PR.